Data_Structures_Answers.md

```diff
@@ -1,18 +1,24 @@
 Add your answers to the questions below.
 
 1. What is the runtime complexity of your ring buffer's `append` method?
+O(1)
 
 2. What is the space complexity of your ring buffer's `append` function?
+O(1)
 
 3. What is the runtime complexity of your ring buffer's `get` method?
+O(n)
 
 4. What is the space complexity of your ring buffer's `get` method?
-
+o(n)
 
 5. What is the runtime complexity of the provided code in `names.py`?
-
+O(n^2)
 6. What is the space complexity of the provided code in `names.py`?
+O(n)
```

Comment:
Hi Manjula! I don't know what you talked about in your 1-1 on Friday but the space complexity of the provided code is O(1)